### ISSUE TITLE
AdaptationClient- Circular Import Issue

### ISSUE THREAD

**Issue by kishoremarodia** [googleapis/python-speech#506]
Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

#### Environment details

  - OS type and version:
  - Python version: 3.9.16
  - pip version: 22.3.1
  - `google-cloud-speech: 2.18.0

#### Code example and steps to reproduce
from google.cloud import speech```

#### Stack trace
ImportError                               Traceback (most recent call last)
~\AppData\Local\Temp\ipykernel_1844\2601755633.py in <cell line: 1>()
----> 1 from google.cloud import speech

~\Anaconda3\lib\site-packages\google\cloud\speech\__init__.py in <module>
     19 
     20 
---> 21 from google.cloud.speech_v1.services.adaptation.client import AdaptationClient
     22 from google.cloud.speech_v1.services.adaptation.async_client import (
     23     AdaptationAsyncClient,

~\Anaconda3\lib\site-packages\google\cloud\speech_v1\services\adaptation\__init__.py in <module>
     14 # limitations under the License.
     15 #
---> 16 from .client import AdaptationClient
     17 from .async_client import AdaptationAsyncClient
     18 

~\Anaconda3\lib\site-packages\google\cloud\speech_v1\services\adaptation\client.py in <module>
     30 )
     31 
---> 32 from google.cloud.speech_v1 import gapic_version as package_version
     33 
     34 from google.api_core import client_options as client_options_lib

~\Anaconda3\lib\site-packages\google\cloud\speech_v1\__init__.py in <module>
     19 
     20 
---> 21 from .services.adaptation import AdaptationClient
     22 from .services.adaptation import AdaptationAsyncClient
     23 from .services.speech import SpeechClient

ImportError: cannot import name 'AdaptationClient' from partially initialized module 'google.cloud.speech_v1.services.adaptation' (most likely due to a circular import) (C:\Users\kmarodia\Anaconda3\lib\site-packages\google\cloud\speech_v1\services\adaptation\__init__.py)

**Comment by Alexi8787** [comment]
hi bro 

It looks like there is an issue with circular imports in the google-cloud-speech library. This can happen when two or more modules try to import each other, causing a circular reference.

One possible workaround is to try importing the speech module from the google.cloud package instead of the google.cloud.speech_v1 package. Here's an example:

python

from google.cloud import speech

client = speech.SpeechClient()

If that doesn't work, you can try upgrading the google-cloud-speech library to the latest version by running:

css

pip install --upgrade google-cloud-speech

If the issue persists, you can try uninstalling and reinstalling the library:

pip uninstall google-cloud-speech
pip install google-cloud-speech

If none of these solutions work, you may want to consider filing a bug report with the library maintainers or reaching out to the Google Cloud support team for further assistance.

**Comment by kishoremarodia** [comment]
Thanks Alex, tried all above steps, didnt help. Looks like some version issues to me. I tried uninstalling and installing these packages too, didnt help

**Comment by dizcology** [comment]
Hello @kishoremarodia : I was not able to reproduce the issue.  Are you able to try installing `google-cloud-speech` in a clean virtual environment?

**Comment by kishoremarodia** [comment]
Was able to resolve the issue.

I already had the packages, reinstalling them also did not solve the issue. I reinstalled whole python distribution and then it sorted out. Also I think I know the reason:

-	This basically needs google-cloud-speech. Before that I had installed many permutations – google, google-cloud, google.cloud and so on. Hence it went into circular loop of searching classes

Thanks a lot for time
